src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const splittedValue = sourceString.split(';');
```

Comment:
The variable name 'splittedValue' is not accurate as it represents multiple values. A more appropriate name would be 'styleEntries' or 'declarations'.